demo/build.gradle

```diff
@@ -8,9 +8,21 @@ buildscript {
 	dependencies {
 		classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
 		classpath('io.spring.gradle:dependency-management-plugin:0.5.4.RELEASE')
+		classpath('mysql:mysql-connector-java:5.1.13')
 	}
 }
 
+plugins {
+	id "org.flywaydb.flyway" version "4.2.0"
+}
+
+flyway {
```

Comment:
besser in der in der application.property abbilden:

https://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html